app/scripts/modules/core/src/pipeline/config/templates/configurePipelineTemplateModal.controller.ts

```diff
@@ -97,6 +109,14 @@ export class ConfigurePipelineTemplateModalController implements IController {
   }
 
   public buildConfig(): IPipelineTemplateConfig {
+    let inheritConfig = [];
```

Comment:
`let` is being flagged by the linter because it's never modified (i.e., use `const` instead). `const` means that the reference can't be changed, but the array the variable points to can still be mutated.

Reply:
👍 I'll fix this up.